### DISCUSSION TITLE
Documentation: more examples of how to use the package

### DISCUSSION THREAD

**Discussion by donmanueldev** [zino-app/graphql-flutter#1210] [Ideas]
@vincenzopalazzo @budde377 

Hello, how are you, the idea of opening this discussion is to know if I can add more examples to the repo, since I have some that I made when I learned to use the package, another is if you can use a well-structured guide using https://docusaurus.io/docs, I think that using this we could have the resources more in order

It would be to start by adding the documentation that is in the readme, then add examples that someone else in the community is encouraged to add

I am willing to start with the first screens, then I will show you advances

**Comment by vincenzopalazzo** [comment]
Please do, maybe in a own repo and we can bring in it

**Reply by donmanueldev** [reply]
ok, very well I'll get to do it

**Reply by donmanueldev** [reply]
<img width="1680" alt="example" src="https://user-images.githubusercontent.com/46093689/186078521-70be2492-fdb4-495f-a769-88dad8397ef7.png">


a preview of how it is going I will continue adding things, then I will do the deploy so you can see the demo 😍

**Reply by donmanueldev** [reply]
<img width="1680" alt="Home" src="https://user-images.githubusercontent.com/46093689/186079235-7cc6fbe6-a1c5-445d-a288-c71a57d35107.png">

**Reply by donmanueldev** [reply]
all parts of the web use markdown to organize components

**Reply by vincenzopalazzo** [reply]
Also take in consideration to take a look to this PR https://github.com/zino-hofmann/graphql-flutter/pull/1205

**Reply by vincenzopalazzo** [reply]
BTW when you have your draft ready we can move inside this https://github.com/graphql-flutter

**Reply by donmanueldev** [reply]
> BTW when you have your draft ready we can move inside this https://github.com/graphql-flutter

ok, it seems great for the moment I will continue adding more things

**Reply by donmanueldev** [reply]
> Also take in consideration to take a look to this PR #1205

Great, I'll take it into account, I was adding the content based on the docs that are in pub dev

**Reply by vincenzopalazzo** [reply]
>a preview of how it is going I will continue adding things, then I will do the deploy so you can see the demo heart_eyes

Do you have the code somewhere? I will try to finish your job if you do not have time

**Reply by donmanueldev** [reply]
Hello how are you, sorry I left everything behind, I got busy at work, what progress is in my profile and if you like we can move it to one side and continue working there or I'll add you to the repository

**Reply by vincenzopalazzo** [reply]
Do not worry, yeah I can add you inside the graphal flutter org and you can move it inside if you like.

What do you think?

**Reply by donmanueldev** [reply]
It's ok, I think it's great
